### PR TITLE
Clean up form validation

### PR DESCRIPTION
**Pull request by julienbourdeau**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | clean up form validation |
| Type? | improvement |
| BC breaks? | no |
| Deprecations? | bootstrap-validation external javascript lib has been removed |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-528 |
| How to test? | Submit weird data into customer address form. submit without required field (edit DOM) |

### CODE REVIEW

File: themes/classic/templates/_partials/form-field.tpl [hunk 1]
```diff
@@ -1,52 +1,91 @@
...
         <input type="{$field.type}" data-minlength="5" data-minlength-error="{l s='Five characters minimum'}" class="form-control" value="{$field.value}" name="{$field.name}" {if $field.required} required {/if}>
+||||||| merged common ancestors
+        <input type="{$field.type}" data-minlength="5" data-minlength-error="{l s='Five characters minimum'}" class="form-control" value="{$field.value}" name="{$field.name}" {if $field.required} required {/if} />
+=======
+
+        <input type="{$field.type}" data-minlength="5" data-minlength-error="{l s='Five characters minimum'}" class="form-control" value="{$field.value}" name="{$field.name}" {if $field.required} required {/if} />
+
+>>>>>>> // FO: customer form Add maxlength contraints
       {else}
+<<<<<<< 31193539ea4cec53e2a80526dc31237c59b16d4b
```
Comment (kpodemski) [inline themes/classic/templates/_partials/form-field.tpl]: oops?

Comment (julienbourdeau) [inline themes/classic/templates/_partials/form-field.tpl]: lol 😅

**Comment by vincentbz** [comment]
Impossible to add a product in the cart : 
impossible produit panier

**Comment by julienbourdeau** [comment]
It should be fixed now :D

**Comment by vincentbz** [comment]
Yep, thank you !

**Pull request merged**